src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -870,5 +870,23 @@ public async Task EvaluatePropertyThatThrows()
                props = await GetObjectOnFrame(frame, "this");
                CheckNumber(props, "a", 11);
            });
+
+        [ConditionalFact(nameof(RunningOnChrome))]
+        public async Task EvaluateMethodAndCheckIsNotPausingOnBreakpoint()
```

Comment:
We should also add tests which do this but with an actual breakpoint.

1.
```csharp
void Foo()
{
   // bp here
}

void Method1()
{
	Method2();
}

void Method2()
{
	... // bp set here
}
```
- set bp in Foo, and Method2
- if we are originally stopped at bp in `Foo`
- then we call `Method1`, we should no break again at Method2

2. And the same thing with a property getter - explicitly invoked, bp in the getter or a method called by that getter
3. And the same thing with a property getter - implicitly invoked because we were evaluating properties on some local var